WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt

```diff
@@ -27,18 +33,23 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
     class PageViewHolder(
         parentView: ViewGroup,
         private val onMenuAction: (PageItem.Action, Page) -> Boolean,
-        private val onItemTapped: (Page) -> Unit
-    ) : PageItemViewHolder(parentView, layout.page_list_item) {
-        private val pageTitle = itemView.findViewById<TextView>(id.page_title)
-        private val pageMore = itemView.findViewById<ImageButton>(id.page_more)
-        private val firstLabel = itemView.findViewById<TextView>(id.first_label)
-        private val secondLabel = itemView.findViewById<TextView>(id.second_label)
-        private val pageItemContainer = itemView.findViewById<ViewGroup>(id.page_item)
-        private val largeStretcher = itemView.findViewById<View>(id.large_stretcher)
-        private val smallStretcher = itemView.findViewById<View>(id.small_stretcher)
+        private val onItemTapped: (Page) -> Unit,
+        private val imageManager: ImageManager,
+        private val isSitePhotonCapable: Boolean
+    ) : PageItemViewHolder(parentView, R.layout.page_list_item) {
+        private val pageTitle = itemView.findViewById<TextView>(R.id.page_title)
+        private val pageMore = itemView.findViewById<ImageButton>(R.id.page_more)
+        private val time = itemView.findViewById<TextView>(R.id.time_posted)
+        private val labels = itemView.findViewById<TextView>(R.id.labels)
+        private val featuredImage = itemView.findViewById<ImageView>(R.id.featured_image)
+        private val pageItemContainer = itemView.findViewById<ViewGroup>(R.id.page_item)
+
+        companion object {
+            const val FEATURED_IMAGE_THUMBNAIL_SIZE_DP = 40
+        }
 
         override fun onBind(pageItem: PageItem) {
-            (pageItem as Page).apply {
+            (pageItem as Page).let {
```

Comment:
It might be worth assigning a name to `it` as it'd get shadowed in the inner closures such as the `map` used to calculate the `labels`. AS also has a warning for this.

_P.S:_ I don't think shadowing in general is a bad idea, Rust does it explicitly, but it does so by invalidating the shadowed value, so it's safe. Unfortunately Kotlin doesn't (can't?) do that. I thought this might be an interesting tidbit to share :)